src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx

```diff
@@ -79,7 +79,7 @@ export const ContentTable = ({rows, isContentNotFound, totalCount, isLoading, is
     }), shallowEqual);
 
     const allowDoubleClickNavigation = nodeType => {
-        return !isStructured && Constants.mode.SEARCH !== mode && (['jnt:folder', 'jnt:contentFolder'].indexOf(nodeType) !== -1);
+        return Constants.mode.SEARCH !== mode && (['jnt:folder', 'jnt:contentFolder'].indexOf(nodeType) !== -1);
```

Comment:
we will be able to double click on structured view even when tree is not displayed, like in folder picker or editorial link picker , or select folder in page accordion , is this really what we want ?

Reply:
Yes, we can navigate folders in those cases as well because of the change and it is probably not what we want to do.